src/main/resources/static/widget.js

```diff
@@ -0,0 +1,36 @@
+function getIframe() {
+  let tcTokenURL = new URL(document.currentScript.src).searchParams.get(
+    "tcTokenURL"
+  );
+  if (tcTokenURL === null) {
+    tcTokenURL = document.getElementById("useid-widget").dataset.tcTokenUrl;
+
+    if (tcTokenURL === undefined) {
+      const error = document.createElement("div");
+      error.innerHTML = "Fehlerhafte Konfiguration: TC Token nicht definiert.";
+      return error;
+    }
+  }
+
+  const iframe = document.createElement("iframe");
+  iframe.setAttribute(
+    "src",
+    `https://useid.dev.ds4g.net/widget?tcTokenURL=${tcTokenURL}`
```

Comment:
The URL should somehow be injected depending on the environment instead of being hard coded? How is the script provided to the customers? Maybe the endpoint should be aware of the environment?

Reply:
Right now, it's only in the static folder. But of course you're right: We shouldn't hard code the URL here, so we probably need some template file and a route, injecting the env?

Reply:
or parse the current URL `window.location.href` and append the query param?

Reply:
The widget will be included in the eService page (window location would point to the eService domain). The URL must point to our backend though.